*** Starting uWSGI 2.0.11.2 (64bit) on [Thu Jan 28 19:07:51 2016] ***
compiled with version: 4.8.2 on 06 December 2015 10:28:08
os: Linux-3.13.0-32-generic #57-Ubuntu SMP Tue Jul 15 03:51:08 UTC 2014
nodename: iZ28zgwg61iZ
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /root/git-lib
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /root/git-lib/Questionnaire
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7770
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 127.0.0.1:8077 fd 3
Python version: 2.7.6 (default, Mar 22 2014, 23:03:41)  [GCC 4.8.2]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x8849e0
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145520 bytes (142 KB) for 2 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x8849e0 pid: 32191 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (pid: 32191, cores: 1)
spawned uWSGI worker 2 (pid: 32194, cores: 1)
[pid: 32191|app: 0|req: 1/1] 171.221.114.138 () {46 vars in 921 bytes} [Thu Jan 28 19:07:58 2016] GET /select/template_1/edit/ => generated 23184 bytes in 39 msecs (HTTP/1.1 200) 2 headers in 88 bytes (1 switches on core 0)
Not Found: /select/Success.html
[pid: 32194|app: 0|req: 1/2] 171.221.114.138 () {54 vars in 1108 bytes} [Thu Jan 28 19:08:36 2016] POST /select/Success.html => generated 2788 bytes in 32 msecs (HTTP/1.1 404) 2 headers in 80 bytes (1 switches on core 0)
*** Starting uWSGI 2.0.11.2 (64bit) on [Thu Jan 28 19:10:58 2016] ***
compiled with version: 4.8.2 on 06 December 2015 10:28:08
os: Linux-3.13.0-32-generic #57-Ubuntu SMP Tue Jul 15 03:51:08 UTC 2014
nodename: iZ28zgwg61iZ
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /root/git-lib
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /root/git-lib/Questionnaire
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7770
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 127.0.0.1:8077 fd 3
Python version: 2.7.6 (default, Mar 22 2014, 23:03:41)  [GCC 4.8.2]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x155d9c0
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145520 bytes (142 KB) for 2 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x155d9c0 pid: 32285 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (pid: 32285, cores: 1)
spawned uWSGI worker 2 (pid: 32288, cores: 1)
[pid: 32285|app: 0|req: 1/1] 171.221.114.138 () {46 vars in 921 bytes} [Thu Jan 28 19:11:05 2016] GET /select/template_1/edit/ => generated 23179 bytes in 37 msecs (HTTP/1.1 200) 2 headers in 88 bytes (1 switches on core 0)
Not Found: /select/success
Internal Server Error: /select/success
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 235, in get_response
    response = middleware_method(request, response)
  File "/usr/local/lib/python2.7/dist-packages/django/middleware/common.py", line 114, in process_response
    return self.response_redirect_class(self.get_full_path_with_slash(request))
  File "/usr/local/lib/python2.7/dist-packages/django/middleware/common.py", line 98, in get_full_path_with_slash
    'url': request.get_host() + new_path,
RuntimeError: You called this URL via POST, but the URL doesn't end in a slash and you have APPEND_SLASH set. Django can't redirect to the slash URL while maintaining POST data. Change your form to point to www.desckie.com/select/success/ (note the trailing slash), or set APPEND_SLASH=False in your Django settings.
[pid: 32288|app: 0|req: 1/2] 171.221.114.138 () {54 vars in 1098 bytes} [Thu Jan 28 19:11:18 2016] POST /select/success => generated 13524 bytes in 65 msecs (HTTP/1.1 500) 1 headers in 64 bytes (1 switches on core 0)
[pid: 32285|app: 0|req: 2/3] 171.221.114.138 () {38 vars in 671 bytes} [Thu Jan 28 19:12:35 2016] GET /select/template_1/edit/ => generated 23179 bytes in 3 msecs (HTTP/1.1 200) 2 headers in 88 bytes (1 switches on core 0)
Not Found: /select/success
Internal Server Error: /select/success
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 235, in get_response
    response = middleware_method(request, response)
  File "/usr/local/lib/python2.7/dist-packages/django/middleware/common.py", line 114, in process_response
    return self.response_redirect_class(self.get_full_path_with_slash(request))
  File "/usr/local/lib/python2.7/dist-packages/django/middleware/common.py", line 98, in get_full_path_with_slash
    'url': request.get_host() + new_path,
RuntimeError: You called this URL via POST, but the URL doesn't end in a slash and you have APPEND_SLASH set. Django can't redirect to the slash URL while maintaining POST data. Change your form to point to www.desckie.com/select/success/ (note the trailing slash), or set APPEND_SLASH=False in your Django settings.
[pid: 32285|app: 0|req: 3/4] 171.221.114.138 () {48 vars in 900 bytes} [Thu Jan 28 19:12:40 2016] POST /select/success => generated 13222 bytes in 34 msecs (HTTP/1.1 500) 1 headers in 64 bytes (1 switches on core 0)
Not Found: /select/success
Internal Server Error: /select/success
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 235, in get_response
    response = middleware_method(request, response)
  File "/usr/local/lib/python2.7/dist-packages/django/middleware/common.py", line 114, in process_response
    return self.response_redirect_class(self.get_full_path_with_slash(request))
  File "/usr/local/lib/python2.7/dist-packages/django/middleware/common.py", line 98, in get_full_path_with_slash
    'url': request.get_host() + new_path,
RuntimeError: You called this URL via POST, but the URL doesn't end in a slash and you have APPEND_SLASH set. Django can't redirect to the slash URL while maintaining POST data. Change your form to point to www.desckie.com/select/success/ (note the trailing slash), or set APPEND_SLASH=False in your Django settings.
[pid: 32288|app: 0|req: 2/5] 171.221.114.138 () {48 vars in 900 bytes} [Thu Jan 28 19:13:39 2016] POST /select/success => generated 13222 bytes in 27 msecs (HTTP/1.1 500) 1 headers in 64 bytes (1 switches on core 0)
[pid: 32285|app: 0|req: 4/6] 123.125.71.73 () {36 vars in 471 bytes} [Thu Jan 28 19:14:23 2016] GET / => generated 218284 bytes in 7 msecs (HTTP/1.1 200) 2 headers in 88 bytes (1 switches on core 0)
